projects/igniteui-angular/src/lib/buttonGroup/buttongroup.component.spec.ts

```diff
@@ -358,6 +360,19 @@ describe('IgxButtonGroup', () => {
         }
     });
 
+    it('should style the corresponding button as deselected when the value bound to the selected input changes', () => {
+        const fixture = TestBed.createComponent(ButtonGroupButtonWithBoundSelectedOutputComponent);
+        fixture.detectChanges();
+        
+        const btnGroupInstance = fixture.componentInstance.buttonGroup;
+
+        fixture.componentInstance.selectedValue = 100;
+        fixture.detectChanges();
+
+        expect(btnGroupInstance.selectedButtons.length).toBe(0);
```

Comment:
To indicate that a change in selection really happened, it is good to verify that initially selectedButtons.lenght is > 0 and buttons[1] is selected.

Reply:
Okay, I've added this suggestion, thank you.